### DISCUSSION TITLE
vsearch --cluster_fast #two ideas

### DISCUSSION THREAD

**Discussion by catnipping** [blex-max/ecto_pipe#2] [Ideas]
What about swapping --cluster_fast with --cluster_size? The first sorts clusters according to seq length, the latter according to seq abundance, which could be more valuable in a OTU analysis. It's handy, but not fundamental.

Second thought: adding the --otutabout command generates a otu table, pretty handy for analysis. --relabel command also relabels the centroid name to whatever it's chosen (e.g. 'OTU_n'). Currently, OTUs are named according to the centroid sequence label (could be confusing if looking at the data after some time).

**Comment by blex-max** [comment]
i) is a good idea, I'll implement that now

ii) I will add the otuabout flag, maybe not relabel for now but you can always fork and add that flag if you like!
